keras/initializers.py

```diff
@@ -35,15 +35,15 @@ class Zeros(Initializer):
     """
 
     def __call__(self, shape, dtype=None):
-        return K.constant(0, shape=shape, dtype=dtype)
+        return K.zeros(shape=shape, dtype=dtype)
```

Comment:
K.zeros returns a variable, which is not necessary here, hence the use of `constant`.